h/views/api_auth.py

```diff
@@ -20,6 +20,7 @@ def access_token(request):
         'access_token': token.value,
         'token_type': 'bearer',
         'expires_in': TOKEN_TTL.total_seconds(),
+        'refresh_token': token.refresh_token,
```

Comment:
Access tokens (any tokens that have an expires, such as those created by the oauth service) always have refresh tokens.

It worries me a little that we only use two types of token (dev tokens with no expires or refresh, and access tokens with both an expires and a refresh) but we're letting some database complexity leak out into services and views (the fact that at the db level, because for efficiency reasons we use a single table for both types of token, the table could contain other combinations, e.g. a token with an expires but no refresh or a refresh but no expires).

The model code is written such that a token always has neither an expires nor a refresh, or it has both. Should the services that use the model be able to assume this? And the oauth service is written so that the tokens it returns always have expires and refresh. Should the views that use the service be able to assume this?

I'm happy to add this check if you still want it though